libs/scripts/scripts.js

```diff
@@ -32,7 +32,7 @@ const locales = {
   pe: { ietf: 'es-PE', tk: 'oln4yqj.css' },
```

Comment:
Line 30 `la: { ietf: 'es-LA', tk: 'oln4yqj.css' },` is completely wrong. The `la` geo is for Latin America, while the `LA` subtag is for Laos. There may be other preexisting mistakes like this.